### PR TITLE
gwpy/cli/timeseries.py: limit sample rate to 3 decimals in plot title

### PR DESCRIPTION
**Pull request by areeda**
I got tired of minute trends reporting Fs to 10 decimal digits.  I verified VIrgo sample rates are not slower than .001 Hz

### CODE REVIEW

File: gwpy/cli/timeseries.py [hunk 1]
```diff
@@ -55,7 +56,9 @@ def get_suptitle(self):
 
     def get_title(self):
         suffix = super(TimeSeries, self).get_title()
-        rates = {ts.sample_rate for ts in self.timeseries}
+        # limit significant digits for minute trends
+        rates = {int((ts.sample_rate.value + 5e-4) * 1000) / 1000. * units.Hz
+                 for ts in self.timeseries}
         fss = '({0})'.format('), ('.join(map(str, rates)))
```
Comment (duncanmmacleod) [inline gwpy/cli/timeseries.py]: @areeda, is there a reason not to use the builtin `round` function here?

File: gwpy/cli/timeseries.py [hunk 2]
```diff
@@ -55,7 +56,9 @@ def get_suptitle(self):
 
     def get_title(self):
         suffix = super(TimeSeries, self).get_title()
-        rates = {ts.sample_rate for ts in self.timeseries}
+        # limit significant digits for minute trends
+        rates = {round(ts.sample_rate.value, 3) * units.Hz
+                 for ts in self.timeseries}
```
Comment (duncanmmacleod) [inline gwpy/cli/timeseries.py]: I'm sorry @areeda, it turns out that Quantities have built-in support for rounding, so can you try using this:

```python
        rates = {ts.sample_rate.round(3) for ts in self.timeseries}
```

Comment (areeda) [reply gwpy/cli/timeseries.py]: How's this/

File: gwpy/cli/timeseries.py [hunk 3]
```diff
@@ -25,6 +25,7 @@
 from .cliproduct import TimeDomainProduct
 from ..plot import Plot
 from ..plot.tex import label_to_latex
+from astropy import units
```
Comment (duncanmmacleod) [inline gwpy/cli/timeseries.py]: @areeda, can you please remove this import, which I don't think is needed anymore

**Pull request merged**